eng/mgmt/AzSdk.test.reference.props

```diff
@@ -12,4 +12,7 @@
 		<DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.1" PrivateAssets="All" />
   </ItemGroup>
+  <ItemGroup> 
+	   <Compile Include="$(RepoRoot)\tools\DisableTestRunParallel.cs" Link="DisableTestRunParallel.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" /> 
```

Comment:
Do you know why the Exclude? It doesn't seem to be related?

Also at a future clean-up time we should move the DisableTestRunParallel.cs under the eng folder.

Reply:
I will do it right now, will move the disable*.cs to eng directory

I beleive this is one of remanants to previous project system, where linking was not working appropriately and had to exclude things. I will try removing it and if eveyrhing works, will remove the exclude clause